src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -3301,110 +3219,89 @@ internal virtual ImmutableArray<LabelSymbol> Labels
+        internal PatternLookupResult FindPatternMethodRelaxed(BoundExpression receiver, string methodName,
+                                                              SyntaxNode syntaxNode, DiagnosticBag diagnostics, out MethodSymbol result)
+                                                       
         {
-            Debug.Assert(lookupResult.IsClear);
+            // PROTOTYPE: try and resolve the method using binding 
+            var bindingDiagnostics = DiagnosticBag.GetInstance();
 
-            HashSet<DiagnosticInfo> useSiteDiagnostics = null;
-            ArrayBuilder<MethodSymbol> candidateMethods = ArrayBuilder<MethodSymbol>.GetInstance();
-            PooledHashSet<NamedTypeSymbol> visitedTypes = null;
-
-            while (!(patternType is null))
+            try
             {
-                this.LookupMembersInType(lookupResult,
-                                         patternType,
-                                         methodName,
-                                         arity: 0,
-                                         basesBeingResolved: null,
-                                         options: LookupOptions.Default,
-                                         originalBinder: this,
-                                         diagnose: false,
-                                         useSiteDiagnostics: ref useSiteDiagnostics);
+                result = null;
 
-                if (lookupResult.IsMultiViable)
-                {
-                    // from the methods we found, see if any are viable
-                    foreach (Symbol member in lookupResult.Symbols)
-                    {
-                        if (member.Kind != SymbolKind.Method)
-                        {
-                            ReportPatternWarning(diagnostics, patternType, member, syntaxExpr, messageID);
-                        }
-
-                        //PROTOTYPE: for now both strict/relaxed lookup require zero params, but relaxed should allow e.g. (params object[])
-                        if (member is MethodSymbol method && !method.Parameters.Any())
-                        {
-                            candidateMethods.Add(method);
-                        }
-                    }
+                var boundAccess = BindInstanceMemberAccess(
+                       syntaxNode,
+                       syntaxNode,
+                       receiver,
+                       methodName,
+                       rightArity: 0,
+                       typeArgumentsSyntax: default,
+                       typeArguments: default,
+                       invoked: true,
+                       indexed: false,
+                       bindingDiagnostics);
+
+                BoundMethodGroup bmg = new BoundMethodGroup(syntaxNode, default, receiver, "Dispose", ImmutableArray<MethodSymbol>.Empty, LookupResult.GetInstance(), BoundMethodGroupFlags.SearchExtensionMethods, false);
```

Comment:
>bmg [](start = 33, length = 3)

Never mind, it looks like `bmg` is not referenced. Let's remove it.

Reply:
Oops. left that in by mistake. Good catch, thanks